tests/console/check_package_version.pm

```diff
@@ -25,11 +25,10 @@ use utils qw(systemctl zypper_call);
 use Mojo::Util 'trim';
 
 my %package = (
-    postgresql12 => '13.0.0',
-    python3      => '3.9.0',
+    postgresql13 => '13.0.0',
+    python39     => '3.9.0',
```

Comment:
should also check python3.6? 
also should we check python2 move to legacy?

Reply:
No need to check python3.6, the file only mentioned the python39. For the python2 and postgresql12, it's another ticket. It checks where the packages moved to, need to install legacy module.

Reply:
There is sentence mention in ppt:
Newer SPs might get newer Python versions in addition, but only 3.6 
and the latest shipped version will be supported simultaneously 
(SP3: 3.9 targeted)

Reply:
Add the checking of python3.6.